test/verify/check-active-pages

```diff
@@ -89,7 +89,10 @@ class TestActivePages(MachineCase):
         # wait until it's highlighted
         b.wait_present('tr.listing-ct-selected[data-row-id="cockpit1:localhost/system"]')
         # close
-        b.click("#active-pages-dialog button.btn-primary")
+        if m.image == "rhel-8-2-distropkg":
+            b.click("#active-pages-dialog button.btn-primary")
+        else:
+            b.click("#active-pages-dialog button.pf-m-primary")
```

Comment:
This could use the new `primary_btn_class`. But no need to re-push just for this.